dasetwas.wings/Survival/CraftingRecipes/craftbot.json

```diff
@@ -0,0 +1,627 @@
+			{
+				"quantity": 3,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat 2x1 Slanted Rounded Edge Major
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a504a",
+		"quantity": 1,
+		"craftTime": 2,
+		"ingredientList": [
+			{
+				"quantity": 4,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat 2x1 Slanted Rounded Edge Minor
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a505a",
+		"quantity": 1,
+		"craftTime": 2,
+		"ingredientList": [
+			{
+				"quantity": 3,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat Curved Rounded Edge Major
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a506a",
+		"quantity": 1,
+		"craftTime": 2,
+		"ingredientList": [
+			{
+				"quantity": 4,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 1x1 Flat Curved Rounded Edge Minor
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a507a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 1x1 Flat Rounded Edge
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a508a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat 90° Curved Rounded Edge
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a509a",
+		"quantity": 1,
+		"craftTime": 2,
+		"ingredientList": [
+			{
+				"quantity": 4,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat 2x1 Slanted Rounded Edge Major with recessed edge
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a510a",
+		"quantity": 1,
+		"craftTime": 2,
+		"ingredientList": [
+			{
+				"quantity": 4,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 1x1 Flat Rounded Edge with extruded edge
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a511a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 1x1 Lil' Nub
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a512a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
+				"itemId": "1f7ac0bb-ad45-4246-9817-59bdf7f7ab39"
+			}
+		]
+	},
+	{
+		// Modular Wing 1x1 Flat Rounded 90° Curve
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a513a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
+				"itemId": "1016cafc-9f6b-40c9-8713-9019d399783f"
+			}
+		]
+	},
+	{
+		// Modular Wing 2x2 Flat 2x1 Slanted Rounded Edge Minor with recessed edge
+		"itemId": "8e3e3ef6-5fc0-4e8a-b07e-398d3d6a514a",
+		"quantity": 1,
+		"craftTime": 1,
+		"ingredientList": [
+			{
+				"quantity": 1,
```

Comment:
is this right?

Reply:
It's a very small piece. It's 2x1, but it's also a shallow slant.

Reply:
ok